cert.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This file should not be added to the repo.

Reply:
Done.